tests/plugins_t/test_elasticsearchmonitoring.py

```diff
@@ -0,0 +1,81 @@
+from tardis.plugins.elasticsearchmonitoring import ElasticsearchMonitoring
+from tardis.utilities.attributedict import AttributeDict
+from tardis.interfaces.siteadapter import ResourceStatus
+from tardis.resources.dronestates import CleanupState
+
+from datetime import datetime
+from unittest import TestCase
+from unittest.mock import Mock
+from unittest.mock import patch
+
+from ..utilities.utilities import run_async
+
+
+class TestElasticsearchMonitoring(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.mock_config_patcher = patch(
+            "tardis.plugins.elasticsearchmonitoring.Configuration"
+        )
+        cls.mock_elasticsearch_patcher = patch(
+            "tardis.plugins.elasticsearchmonitoring.Elasticsearch"
+        )
+        cls.mock_datetime_patcher = patch(
+            "tardis.plugins.elasticsearchmonitoring.datetime"
+        )
+        cls.mock_time_patcher = patch("tardis.plugins.elasticsearchmonitoring.time")
+        cls.mock_config = cls.mock_config_patcher.start()
+        cls.mock_elasticsearch = cls.mock_elasticsearch_patcher.start()
+        cls.mock_datetime = cls.mock_datetime_patcher.start()
+        cls.mock_time = cls.mock_time_patcher.start()
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.mock_config_patcher.stop()
+        cls.mock_elasticsearch_patcher.stop()
+        cls.mock_datetime_patcher.stop()
+        cls.mock_time_patcher.stop()
+
+    @patch("tardis.plugins.elasticsearchmonitoring.logging", Mock())
+    def setUp(self):
+        self.config = self.mock_config.return_value
+        self.config.Plugins.ElasticsearchMonitoring._index = "index"
+        self.config.Plugins.ElasticsearchMonitoring._meta = "meta"
+
+        self.plugin = ElasticsearchMonitoring()
+
+    @patch("tardis.plugins.elasticsearchmonitoring.logging", Mock())
+    def test_notify(self):
+        test_param = AttributeDict(
+            site_name="test-site",
+            machine_type="test_machine_type",
+            created=datetime.now(),
+            updated=datetime.now(),
+            resource_status=ResourceStatus.Booting,
+            drone_uuid="test-drone",
+        )
+
+        test_param_ext = {
+            **test_param,
+            "state": str(CleanupState()),
+            "meta": self.plugin._meta,
+            "timestamp": int(self.mock_time.return_value * 1000),
```

Comment:
`self.mock_time.return_value` should be set to a reasonable value before using it.